lib/appsignal.rb

```diff
@@ -324,4 +324,5 @@ def collect_environment_metadata
 require "appsignal/transaction"
 require "appsignal/version"
 require "appsignal/rack/generic_instrumentation"
+require "appsignal/span"
```

Comment:
Do we want to load it by default for everyone? If it's experimental, and not used by our users for now, should we not make it optional. We can manually require it in the agent test suite for now.

Reply:
That makes sense, will change that for the moment.